t/t2205-job-info-security.t

```diff
@@ -8,31 +8,30 @@ test_under_flux 4 job
 
 # we have to fake a job submission by a guest into the KVS.
 
-# it is not 100% clear the most portable way to add a newline to a
-# string, so I do a classic trick of adding an extra char at the end
-# to avoid newline stripping, then strip the extra char.
-
+# Usage: submit_job [userid]
```

Comment:
minor nit, perhaps add a comment saying why you do the get --raw | sed | put --raw, it's b/c you want to preserve the trailing newline.

Reply:
Agreed!   I added this:
```
# This method of editing the eventlog preserves newline separators.
```